drivers/gles3/storage/render_scene_buffers_gles3.cpp

```diff
@@ -62,24 +79,43 @@ void RenderSceneBuffersGLES3::configure(RID p_render_target, const Size2i p_inte
 	glGenFramebuffers(1, &framebuffer);
 	glBindFramebuffer(GL_FRAMEBUFFER, framebuffer);
 
-	glBindTexture(GL_TEXTURE_2D, rt->color);
-	glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D, rt->color, 0);
+	if (view_count > 1 && config->multiview_supported) {
+		glBindTexture(GL_TEXTURE_2D_ARRAY, rt->color);
+		glFramebufferTextureMultiviewOVR(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, rt->color, 0, 0, view_count);
```

Comment:
This will fail on devices that don't expose ``glFramebufferTextureMultiviewOVR`` and where ``!defined(GLES_OVER_GL) && !defined(WEB_ENABLED)`` is false. These blocks need to be behind the same ``#if !defined(GLES_OVER_GL) && !defined(WEB_ENABLED)`` statements as the initialization of ``glFramebufferTextureMultiviewOVR``

Reply:
Hm, I'm not sure this is right.

The `glFramebufferTextureMultiviewOVR()` function is loaded in one of three ways:

1. Via GLAD on desktop OpenGL
2. Via godot_webgl2.h on WebGL
3. Via the `eglGetProcAddress()` that you commented on above, on real GLES (which I think might only be Android?)

It's only nr 3 that's wrapped in `#if !defined(GLES_OVER_GL) && !defined(WEB_ENABLED)`, but we need that code to run in case nr 1 and 2 as well

Reply:
Okay, as long as ``glFramebufferTextureMultiviewOVR`` is defined somewhere in all cases we should be good.